test/suites/vm.sh

```diff
@@ -65,3 +69,48 @@ test_vm_empty() {
   ! lxc start v1 || false
   lxc delete v1
 }
+
+test_snap_vm_empty() {
```

Comment:
this doesnt need to be a specific snap test does it, it seems to be this would be preferable as a non-snap test?